apps/next/src/pages/AccountManagement/components/Wallets/components/BulkDeleteButtons.tsx

```diff
@@ -72,6 +76,24 @@ export const BulkDeleteButtons: FC = () => {
           </Button>
         </Stack>
       </Slide>
-    </>
+    </BulkDeleteButtonsContainer>
   );
 };
+
+const BulkDeleteButtonsContainer = styled(Stack)(({ theme }) => ({
+  position: 'sticky',
+  bottom: 0,
+  zIndex: 10,
+  paddingTop: theme.spacing(1),
+  paddingBottom: theme.spacing(1.5),
+  marginLeft: `-${theme.spacing(2)}`,
+  paddingInline: theme.spacing(2),
+  marginRight: `-${theme.spacing(2)}`,
+  marginBottom: `-${theme.spacing(1.5)}`,
```

Comment:
These negative margins on the side are causing the content to scroll horizontally:

<img width="339" height="628" alt="Image" src="https://github.com/user-attachments/assets/2e3a80b3-6b6f-40b8-95ac-71865f3f5203" />